### ISSUE TITLE
train.py

### ISSUE THREAD

**Issue by Blues5** [xiadingZ/video-caption-pytorch#2]
if not sc_flag:if not sc_flag:
                seq_probs = model(fc_feats, labels)
                loss = loss_fn(seq_probs, labels[:, 1:], masks[:, 1:])
check this.

**Comment by xiadingZ** [comment]
when calculating loss, doesn't count the `<sos>` token of labels, so use `labels[:, 1:]` and `masks[:, 1:]`
